CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
```

Comment:
Nothing in this list should preclude using the standard library, and usage of anything in this list is not required. The reason I had for being explicit about third-party dependencies is to add a little strategic friction. For something to be included in `simpl` we prefer using libraries we have already agreed upon.

Maybe including this is too prescriptive? It didn't feel like it when I wrote it, but I'm willing to consider it.

Should we remove `arrow` from this list? I'm okay with that. Does that mean `arrow` shouldn't be used? I'm less okay with that, but no completely opposed to it.